care/facility/api/serializers/patient.py

```
@@ -210,6 +211,11 @@ def validate_year_of_birth(self, value):
             raise serializers.ValidationError(msg)
         return value
 
+    def validate_name(self, value):
+        if not re.match(r'^[a-zA-Z\s]+$', value):
```

Ruff

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)